adagrams/game.py

```diff
@@ -1,13 +1,111 @@
+# comment to practice push to GitHub
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+POINTS = {
+    1:["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"],
+    2:["D","G"],
+    3:["B","C","M","P"],
+    4:["F","H","V","W","Y"],
+    5:["K"],
+    8:["J","X"],
+    10:["Q","Z"]
+    }
+
 def draw_letters():
-    pass
+    total_letter_pool = []
+    hand = []
+
+    # create a list with the total amount of letters: 
+    for letter, quantity in LETTER_POOL.items():
+        total_letter_pool.extend([letter for i in range(quantity)])
```

Comment:
👍 Since we declare `i` but don't access it, prefer to use `_` instead, like: 

```suggestion
    for letter, quantity in LETTER_POOL.items():
        total_letter_pool.extend([letter for _ in range(quantity)])
```

Reply:
Prefer to have this logic for creating a total letter pool in a helper function like `create_letter_pool`. Then you can call that helper function in `draw_letters` which would make it more concise and more single responsibility (instead of needing to create a pool and draw letters)